inspirehep/dojson/conferences/fields/bd1xx.py

```diff
@@ -33,40 +35,70 @@
 @utils.for_each_value
 def acronym(self, key, value):
     """Conference acronym."""
+    def append_to_self(title):
+        """Make a dictionary and append to self."""
+        titles = {}
+        titles['title'] = title
+        titles['subtitle'] = value.get('b')
+        titles['source'] = value.get('9')
+        if titles:
+            self['titles'].append(titles)
+
     self['date'] = value.get('d')
     self['opening_date'] = value.get('x')
     self['closing_date'] = value.get('y')
     self['cnum'] = value.get('g')
-    self['place'] = value.get('c')
-    self['subtitle'] = value.get('b')
-    self['title'] = value.get('a')
+    self.setdefault('titles', [])
+    title = value.get('a')
+    if isinstance(title, (tuple, list)):
+        for t in title:
+            append_to_self(t)
+    else:
+        append_to_self(title)
+
+    if value.get('c'):
+        self.setdefault('address', [])
+        raw_addresses = utils.force_list(value.get('c'))
+        for raw_address in raw_addresses:
+            address = inspire_dojson_utils.parse_conference_address(raw_address)
+            self['address'].append(address)
+
     return utils.force_list(value.get('e'))
 
 
 @conferences.over('alternative_titles', '^711')
 @utils.for_each_value
 def alternative_titles(self, key, value):
-    """Alternative title."""
-    return value.get('a')
+    """Alternative title.
 
-
-@conferences.over('contact_person', '^270')
-@utils.for_each_value
-def contact_person(self, key, value):
-    """Contact person."""
-    if value.get('m'):
-        self['contact_email'] = value.get('m')
-    return value.get('p')
+    711__b is for indexing and will not be displayed.
+    """
+    titles = {}
+    titles['title'] = value.get('a')
+    titles['searchable_title'] = value.get('b')
+    return titles
 
 
-@conferences.over('field_code', '^65017')
+@conferences.over('contact_details', '^270')
 @utils.for_each_value
 @utils.filter_values
-def field_code(self, key, value):
-    """Field code."""
+def contact_details(self, key, value):
+    """Map 270 field to contact details and extra_place_info"""
+    self.setdefault('extra_place_info', [])
+    extra_place_info = value.get('b')
+    if isinstance(extra_place_info, six.string_types):
+        self['extra_place_info'].append(extra_place_info)
+    elif isinstance(extra_place_info, tuple):
+        for place in extra_place_info:
+            self['extra_place_info'].append(place)
+    else:
+        pass
```

Comment:
Writing
```python
else:
    pass
```
is equivalent to writing nothing, so those cases are still unhandled...